src/views/vehicleServiceDetails/vehicleServicepage.jsx

```diff
@@ -1,429 +1,455 @@
-                    <th scope='col'>Dummy</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <tr>
-                    <th scope='row'>Dummy</th>
-                    <td>Dummy</td>
-                  </tr>
-                </tbody>
-              </Table>
-            </div>
-            <div className='modal-footer'>
-              <Button
-                className='ml-auto'
-                color='link'
-                data-dismiss='modal'
-                type='button'
-                onClick={() => this.toggleModal('defaultModal')}
-              >
-                Close
-              </Button>
-            </div>
-          </Modal>
-        </Container>
-      </>
-    );
-  }
-}
-
-export default vehicleServicepage;
+import React, { Component, Fragment } from 'react';
+import 'react-table/react-table.css';
+import ReactTable from 'react-table';
+import { connect } from 'react-redux';
+// reactstrap components
+import {
+  Card,
+  Container,
+  Row,
+  Col,
+  ListGroup,
+  ListGroupItem,
+  Button,
+  DropdownMenu,
+  DropdownItem,
+  UncontrolledDropdown,
+  DropdownToggle,
+  Table,
+  Modal
+} from 'reactstrap';
+import UserHeader from 'components/Headers/UserHeader.jsx';
+import { vehicleServiceDetails } from '../../redux/actions/Index';
+import {
+  withScriptjs,
+  withGoogleMap,
+  GoogleMap,
+  Marker
+} from 'react-google-maps';
+
+const MapWrapper = withScriptjs(
+  withGoogleMap(props => (
+    <GoogleMap
+      defaultZoom={12}
+      defaultCenter={{ lat: 40.748817, lng: -73.985428 }}
+      defaultOptions={{
+        scrollwheel: false,
+        styles: [
+          {
+            featureType: 'administrative',
+            elementType: 'labels.text.fill',
+            stylers: [{ color: '#444444' }]
+          },
+          {
+            featureType: 'landscape',
+            elementType: 'all',
+            stylers: [{ color: '#f2f2f2' }]
+          },
+          {
+            featureType: 'poi',
+            elementType: 'all',
+            stylers: [{ visibility: 'off' }]
+          },
+          {
+            featureType: 'road',
+            elementType: 'all',
+            stylers: [{ saturation: -100 }, { lightness: 45 }]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'all',
+            stylers: [{ visibility: 'simplified' }]
+          },
+          {
+            featureType: 'road.arterial',
+            elementType: 'labels.icon',
+            stylers: [{ visibility: 'off' }]
+          },
+          {
+            featureType: 'transit',
+            elementType: 'all',
+            stylers: [{ visibility: 'off' }]
+          },
+          {
+            featureType: 'water',
+            elementType: 'all',
+            stylers: [{ color: '#5e72e4' }, { visibility: 'on' }]
+          }
+        ]
+      }}
+    >
+      <Marker position={{ lat: 40.748817, lng: -73.985428 }} />
```

Comment:
maintain default lat long in constant file, reuse here. 
same is used above too